src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java

```diff
@@ -51,9 +51,6 @@ public JavaSchemaTypeNameConverter(String implicitPackageName, JavaNameFormatter
   }
 
   private static String getPrimitiveTypeName(Schema schema) {
-    if (schema == null) {
```

Comment:
this was never called (also not called in production, when generating the compute API)